cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2497,34 +2489,13 @@ Slice::Gen::DataDefVisitor::visitClassDefStart(const ClassDefPtr& p)
     H << nl << "[[nodiscard]] " << name << "Ptr ice_clone() const { return std::static_pointer_cast<" << name
       << ">(_iceCloneImpl()); }";
 
-    return true;
-}
-
-void
-Slice::Gen::DataDefVisitor::visitClassDefEnd(const ClassDefPtr& p)
-{
-    const string name = p->mappedName();
-    const string scoped = p->mappedScoped();
-    const string scope = p->mappedScope();
-    const ClassDefPtr base = p->base();
-
-    const DataMemberList dataMembers = p->dataMembers();
-
-    H << sp;
-    for (const auto& dataMember : dataMembers)
-    {
-        emitDataMember(dataMember);
-    }
-
     if (p->hasMetadata("cpp:custom-print"))
```

Comment:
Moved this code to visitClassDefStart to be next to the other public member functions.